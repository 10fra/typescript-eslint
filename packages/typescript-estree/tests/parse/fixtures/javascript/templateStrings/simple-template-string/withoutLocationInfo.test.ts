import path from 'path';
import { testWithoutLocation } from 'test-fixture';

testWithoutLocation(
  path.resolve(
    process.cwd(),
    '..',
    'shared-fixtures',
    'fixtures/javascript/templateStrings/simple-template-string.src.js',
  ),
  {
    useJSXTextNode: false,
  },
);
