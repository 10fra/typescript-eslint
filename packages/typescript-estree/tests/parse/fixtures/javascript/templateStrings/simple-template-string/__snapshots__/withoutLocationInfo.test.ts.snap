// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`/Users/bradzacher/github/typescript-eslint/packages/shared-fixtures/fixtures/javascript/templateStrings/simple-template-string.src.js parses without location information 1`] = `
Object {
  "body": Array [
    Object {
      "expression": Object {
        "expressions": Array [],
        "loc": Object {
          "end": Object {
            "column": 4,
            "line": 1,
          },
          "start": Object {
            "column": 0,
            "line": 1,
          },
        },
        "quasis": Array [
          Object {
            "loc": Object {
              "end": Object {
                "column": 4,
                "line": 1,
              },
              "start": Object {
                "column": 0,
                "line": 1,
              },
            },
            "range": Array [
              0,
              4,
            ],
            "tail": true,
            "type": "TemplateElement",
            "value": Object {
              "cooked": "42",
              "raw": "42",
            },
          },
        ],
        "range": Array [
          0,
          4,
        ],
        "type": "TemplateLiteral",
      },
      "loc": Object {
        "end": Object {
          "column": 5,
          "line": 1,
        },
        "start": Object {
          "column": 0,
          "line": 1,
        },
      },
      "range": Array [
        0,
        5,
      ],
      "type": "ExpressionStatement",
    },
  ],
  "comments": Array [],
  "loc": Object {
    "end": Object {
      "column": 5,
      "line": 1,
    },
    "start": Object {
      "column": 0,
      "line": 1,
    },
  },
  "range": Array [
    0,
    5,
  ],
  "sourceType": "script",
  "tokens": Array [
    Object {
      "loc": Object {
        "end": Object {
          "column": 4,
          "line": 1,
        },
        "start": Object {
          "column": 0,
          "line": 1,
        },
      },
      "range": Array [
        0,
        4,
      ],
      "type": "Template",
      "value": "\`42\`",
    },
    Object {
      "loc": Object {
        "end": Object {
          "column": 5,
          "line": 1,
        },
        "start": Object {
          "column": 4,
          "line": 1,
        },
      },
      "range": Array [
        4,
        5,
      ],
      "type": "Punctuator",
      "value": ";",
    },
  ],
  "type": "Program",
}
`;
